client/landing/jetpack-cloud/sections.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
+	{
+		name: 'jetpack-cloud',
+		paths: [ '/', '/scan', '/backups', '/settings' ],
+		module: 'landing/jetpack-cloud',
```

Comment:
Feels weird to have a section that lives there. If this is now a section in the main entry point, let's move it to `client/jetpack-cloud` instead of in `client/landing/jetpack-cloud`. Rationale: `landing` currently holds only the separate entry points.

Reply:
That is a pretty big change I would rather we do it in a different PR. Since it would also involve components being moved to the components folder etc and I would also like to give the team abit of a heads up. So that we don't end up in a state where most things get need a rebase.